### PR TITLE
Check in/85599/card select

### PR DESCRIPTION
**Pull request by brianseek**
## Summary

Updates the OH travel only app content around appointment selection and display.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#85599
department-of-veterans-affairs/va.gov-team#85597
department-of-veterans-affairs/va.gov-team#85600

## Testing done

Visual, updates unit, and e2e

## What areas of the site does it impact?

Travel claim application for Oracle Health appointments

## Acceptance criteria
 - [ ] The application matches the wires linked in the ticket.
 
### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx [hunk 1]
```diff
@@ -12,16 +17,29 @@ const TravelClaimSuccessAlert = () => {
...
         <p
           className="vads-u-margin-top--0"
           data-testid="travel-pay--claim--submitted"
         >
-          {`${t('this-claim-is-for-your-appointment', {
-            date: format(new Date(), 'MMMM dd, yyyy'),
-            claims: 1,
-            appointments: 1,
-          })} ${t('well-send-you-a-text-to-let-you-know', { count: 1 })}`}
+          {`${t('this-claim-is-for-your-appointment')} ${t('appointment-on', {
```
Comment (wullaski) [inline src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx:25]: I thought this should read `...for your in person appointment` but I could be confused.

File: src/applications/check-in/travel-claim/pages/travel-review/index.jsx [hunk 2]
```diff
@@ -50,11 +51,25 @@ const TravelReview = props => {
         <dt className="vads-u-margin-top--2p5">{t('what-youre-claiming')}</dt>
         <dd className="vads-u-margin-top--0p5">
           <span data-testid="claim-info">
-            {`${t('mileage-only-reimbursement-for')} ${
-              appointmentToFile.clinicStopCodeName
-                ? ` ${appointmentToFile.clinicStopCodeName}`
-                : ` ${t('VA-appointment')}`
-            } at ${appointmentToFile.startTime}`}
+            {`${t('mileage-only-reimbursement-for-your')} ${t(
```
Comment (wullaski) [inline src/applications/check-in/travel-claim/pages/travel-review/index.jsx:54]: Same as the other comment above. I'm not sure if this is supposed to say `in person` or not.

**Review by wullaski — DISMISSED** [review]
Looks good! I'm just confused about some of the content around the appointment type

File: src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx (hunk 1 above)
Comment (brianseek) [reply src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx:25]: no, you are correct, I missed that update in the Figma. I'll get it re-worked.

File: src/applications/check-in/travel-claim/pages/travel-review/index.jsx (hunk 2 above)
Comment (brianseek) [reply src/applications/check-in/travel-claim/pages/travel-review/index.jsx:54]: Correct again, good catches!

**Review by wullaski — APPROVED** [review]
LGTM!

**Review by CBonade — APPROVED** [review]

**Pull request merged**